frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -227,7 +250,12 @@ function FileSearcher:showSearchResultsMessage(no_results)
     end
 end
 
-function FileSearcher:showSearchResults(results)
+function FileSearcher:onShowSearchResults(not_cached)
+    if (not not_cached and self.search_results == nil) then
```

Comment:
Parens seems unnecessary (seeing them, it feels you may have made a typo and wanted to have the `not` outside).